admin/includes/modules/product/preview_info.php

```diff
@@ -89,10 +89,11 @@
           $products_image_name = $_POST['img_dir'] . $_POST['products_image_manual'];
           $pInfo->products_name = $products_image_name;
         }
+        $display_products_description = str_replace('images/', DIR_WS_CATALOG_IMAGES, $display_products_description);
```

Comment:
This line should be changed to
```php
$display_products_description = str_replace('src="images/', 'src="' . DIR_WS_CATALOG_IMAGES, $pInfo->products_description);
```

- Correcting the source-variable for the `str_replace`
- Limiting the replacement to the `src="` attribute of an `<img>` tag.  Otherwise, a fully-specified `https://mysite.com/images/image_name.jpg` would have _unwanted_ replacements.

Note that this change also needs to be applied to the like-named .php file for the `document_general`, `document_product`, `product_free_shipping` and `product_music` types.

Reply:
May I also propose moving this new line 92 down to line 97/98 after the `if/else` as a new `echo` statement?

This is because this "description" var is being output in both cases, so isn't really part of the if/else.